MFVNeutralino/plugins/VertexTracks.cc

```diff
@@ -1110,11 +1105,10 @@ bool MFVVertexTracks::filter(edm::Event& event, const edm::EventSetup& setup) {
     }
     for (size_t i = 0, ie = all_electron_tracks->size(); i < ie; ++i) {
       const reco::TrackRef& etk = (*all_electron_tracks)[i];
-      //this will need to be changed once we have a different(?) track rescaler for leptons ... ?
-      const auto rs = track_rescaler.scale(*etk);
-
+      const auto rs = (track_rescaler_which == 1) ? track_rescaler.scale(*etk, "electron")
```

Comment:
@abbywarden see comment before about muons and the track rescaler index vs enum value